### ISSUE TITLE
PSU connector reversed

### ISSUE THREAD

**Issue by Biondilbiondo** [gerb-ster/TH-555-VCO#1]
Hi, the PSU connector is reversed wrt the eurorack standard. Pin 1 should be -12 V and pin 10 +12 V (https://division-6.com/learn/eurorack-power/ and https://doepfer.de/a100_man/a100t_e.htm).
In your schematic and on your website (https://www.gerbster.nl/wp-content/uploads/2021/02/IMG_2688-scaled.jpeg) it seems that pin 1 should be +12 V and pin 10 -12 V. This is easily worked around by mounting the connector with the hole looking at the interior of the PCB, so in the opposite way respect to what is marked on the silkscreen.
I was extremely cautious, and I double-checked everything before connecting to the PSU but someone can burn all the chips and explode the two electrolytic capacitors due to this error. Please state this clearly and loud in the readme.md !

**Comment by kristjem** [comment]
@Biondilbiondo have you successfully built this module, and does it work as expected if you rotate the 10 pin header connector 180 degrees? I'm considering ordering PCBs.

**Comment by Biondilbiondo** [comment]
Yes it works with the connector reversed. If you are interested in the module write me, I have some spare PCBs which I'm happy to send you to avoid to waste them...

**Comment by kristjem** [comment]
> Yes it works with the connector reversed. If you are interested in the module write me, I have some spare PCBs which I'm happy to send you to avoid to waste them...

Brilliant! Thanks, I've sent you an email.

**Comment by AuguB** [comment]
Hi @Biondilbiondo , do you happen to have some more space pcb's? Otherwise I will place an order myself.

**Comment by gerb-ster** [comment]
I think so, drop me an email at gerb@gerbster.nl and I'll let you know!

**Comment by Biondilbiondo** [comment]
I do have a last set of PCBs, if you need it, I can send you for free. In case you're interested, just write me.

**Comment by 8lall0** [comment]
Hi @Biondilbiondo , i'll be interested too!

**Comment by Biondilbiondo** [comment]
My spear parts are over sorry.

**Comment by canonofd** [comment]
Hi. I’m planning to build this VCO module, and to be honest, I was wondering if you're happy with it. Apart from the issue you mentioned, did you run into any other problems?

**Comment by kristjem** [comment]
> Hi. I’m planning to build this VCO module, and to be honest, I was wondering if you're happy with it. Apart from the issue you mentioned, did you run into any other problems?
> 

Hi! I'm very pleased with the VCO, it's very good sounding and versatile. I sometimes play it by hand by tuning the "fine" pot. The VCO has a huge range, and can somehow be used as a LFO. I built mine with a normal resistor in stead of a thermoistor, and it has been working fine, but I don't move around and my synth is in a heated/balanced room. Only thing I miss is an octave switch, especially since the "course" pot is so sensitive/huge throws.

**Comment by Biondilbiondo** [comment]
Here https://github.com/Biondilbiondo/Eurorack-TH555VCO-Suboctave you can find my version on this very same module. It adds an octave switch and a double sub-octave module. Unfortunately I haven't tested the 0.3 version which solves several problem I had in the previous ones. But if you are a bit into electronics take a look, it is functional and with minor tweaks it works well.
